validator/client/runner.go

```diff
@@ -52,9 +53,15 @@ func run(ctx context.Context, v Validator) {
 			return // Exit if context is cancelled.
 		case slot := <-v.NextSlot():
 			span.AddAttributes(trace.Int64Attribute("slot", int64(slot)))
+			// Report this validator client's rewards and penalties throughout its lifecycle.
+			if err := v.ReportValidatorPerformance(ctx, slot); err != nil {
```

Comment:
Can we defer this to prometheus metrics? Printing to a log is OK, but it might be nicer to run a dashboard as a sidecar that reads the prometheus metrics

Reply:
I want our validator client to be more useful to people running it in the testnet than just slot logs. Having something that logs nicely your rewards over time would be super useful IMO and a major improvement than the current logs.